nengo/builder/operator.py

```diff
@@ -386,6 +386,10 @@ def step_simpyfunc():
                 if y is None:
                     raise ValueError(
                         "Function '%s' returned invalid value" % fn.__name__)
```

Comment:
Maybe we should change this message too, to say that the invalid value here is `None`. I was trying to think if we could get away without this block, but we still need it because Numpy casts None to NaN, annoyingly.